samples/bluetooth/mesh/sensor_client/src/model_handler.c

```diff
@@ -23,15 +23,15 @@ static void sensor_cli_data_cb(struct bt_mesh_sensor_cli *cli,
 		if (value->val1) {
 			printk("Presence detected\n");
 		} else {
-			printk("End of presence\n");
+			printk("No presence detected\n");
 		}
 	} else if (sensor->id ==
 		   bt_mesh_sensor_time_since_presence_detected.id) {
 		if (value->val1) {
 			printk("%s second(s) since last presence detected\n",
 			       bt_mesh_sensor_ch_str(value));
 		} else {
-			printk("No presence detected, or under 1 second since presence detected\n");
+			printk("Presence detected, or under 1 second since presence detected\n");
```

Comment:
I think the previous message was more correct, because `prev_pres` is 0 at the start when no presence is detected yet.

Reply:
prev_pres is set at start, so it is not 0 at start. So it will start reporting last presence detected right away.
The other thing is that it is strange to say "No presence detected,.." when there is actual a presence there. As long it is registered a presence this value can be sent, as it is possible to have a longer presence now. (Long as the user hold the button there is a presence.)

Reply:
Ok, but is it correct to send that presence was detected at the start?

Reply:
Before it is started is no way to say if presence or not exist. So it could be argued both ways. In my view this is the most correct way at least at startup. It says something about time since last presence is detected or it could have been a presence before startup.